Modules/Scripted/ExtensionWizard/ExtensionWizardLib/ModuleInfo.py

```diff
@@ -44,6 +48,24 @@ def findModules(path, depth):
             # __init__.py is not a module but an embedded Python library
             # that a module will load.
```

Comment:
Allowing the following code to be used in `qSlicerScriptedLoadableModuleFactory::isValidFile` would be a nice improvement.

As a first step, introducing `slicer.util.isScriptedLoadableModule` (located after `slicer.util.importModuleObjects`) may be sensible. (somewhat equivalent to `qSlicerUtils::isLoadableModule`)

Since this is not directly relevant to this pull request, I will create a dedicated issue to track this.

Reply:
OK.